src/Task/Base/ParallelExec.php

```diff
@@ -127,6 +146,9 @@ public function run()
             $process->setTimeout($this->timeout);
             $process->start();
             $this->printTaskInfo($process->getCommandLine());
+            if ($this->waitInterval) {
+                sleep($this->waitInterval);
```

Comment:
Bonus points: if the queue is empty, sleep for a short period of time, so that we aren't cranking through the `while` loop too hot.